src/llvm/analysis/ReachingDefinitions/LLVMRDBuilderDense.h

```diff
@@ -0,0 +1,99 @@
+#ifndef _LLVM_DG_RD_DENSE_H_
+#define _LLVM_DG_RD_DENSE_H_
+
+#include <vector>
+#include <unordered_map>
+
+#include <llvm/Support/raw_os_ostream.h>
+#include <llvm/IR/Instructions.h>
+#include <llvm/IR/Constants.h>
+
+#include "llvm/MemAllocationFuncs.h"
+#include "analysis/ReachingDefinitions/ReachingDefinitions.h"
+#include "llvm/analysis/ReachingDefinitions/LLVMRDBuilder.h"
+#include "llvm/analysis/PointsTo/PointsTo.h"
+
+namespace dg {
+namespace analysis {
+namespace rd {
+
+class LLVMRDBuilderDense : public LLVMRDBuilder {
+private:
+    struct Subgraph {
```

Comment:
Isn't this inherited from `LLVMRDBuilder`?

Reply:
Yes, they are. They are unnecessary here, so I'll get rid of them.